reuse_test.go

```diff
@@ -1,45 +1,65 @@
 package reuseport
 
 import (
-	"context"
-	"net"
-	"strings"
 	"testing"
-
-	"github.com/stretchr/testify/require"
 )
 
-func testDialFromListeningPort(t *testing.T, network, host string) {
```

Comment:
Fully removed the previous tests because they were outdated and some of their logic had been moved to `interface.go`.

Reply:
I think the new tests are better because they exercise the code via the public API. Namely I directly call `Listen(network, address string)`, `ListenPacket(network, address string)`, ` Dial(network, laddr, raddr string)` and `Available()`. Instead the current tests instantiate their own `ListenConfig` and call methods on that instead of via the wrappers offered in `interface.go`. For this reason a large part of the code does not get exercised.

Instead of re-writing the tests, I'd be happy to rework the current `testDialFromListeningPort` method to make the right calls.